code/neuron_model.py

```diff
@@ -35,12 +46,12 @@
           'A2': 0.0,             # I₂ additive update constant
           'G': 0.05,             # Membrane potential decay term
           'C': 1.0,              # Membrane capacitance
-          'Iext': np.zeros((200,)),    # External current
           'ThetaInf': -50.0}     # Reverse threshold
 
 
 # @profile
-def mnn_model(pms=params, time=10, dt=1.0, IC=(0.01, 0.001, -70.0, -50.0)):
+def mnn_model(pms=params, Iext=np.zeros((200,)), dt=1.0,
```

Comment:
This is weird. In my last code update, the issue with the mutable has been fixed.